### PR TITLE
con-3583

### PR DESCRIPTION
**Pull request by hkf-tech**
Redirect to PC cohort details

### CODE REVIEW

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs [hunk 1]
```diff
@@ -56,7 +56,7 @@ public async Task<ActionResult> AgreementNotSigned(long providerId, string hashe
             model.RequestListUrl = Url.Action(redirectTo, new { providerId });
 
             if (model.IsSignedAgreement)
-                return RedirectToAction("Details", new { providerId, hashedCommitmentId });
+                return Redirect(_providerUrlhelper.ProviderCommitmentsLink($"{providerId}/unapproved/{hashedCommitmentId}/details"));
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs:59]: Do we have anything in the backlog to move this controller action over to das-providercommitments?

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs [hunk 2]
```diff
@@ -93,28 +93,11 @@ public ActionResult DraftList(long providerId)
...
-
-        private void LogUserClaims()
-        {
-            var claims = ((ClaimsIdentity)HttpContext.User.Identity).Claims
-                        .Select(x => $"{x.Type}: {x.Value}").ToArray();
-
-            var logValue = string.Join(Environment.NewLine, claims);
-
-            _logger.Trace("User claims", new Dictionary<string, object> { { "providerClaims", logValue } });
+            return Redirect(_providerUrlhelper.ProviderCommitmentsLink($"{providerId}/unapproved/{hashedCommitmentId}/details"));
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs:100]: should we be logging before these re-directs similar das-employercommitments?

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs [hunk 3]
```diff
@@ -142,8 +125,9 @@ public async Task<ActionResult> DeleteCohort(DeleteCommitmentViewModel viewModel
             }
 
             if (viewModel.DeleteConfirmed == null || !viewModel.DeleteConfirmed.Value)
-            {   
-                return RedirectToAction("Details", new { providerId = viewModel.ProviderId, hashedCommitmentId = viewModel.HashedCommitmentId });
+            {
+
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs:129]: blank space

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs [hunk 4]
```diff
@@ -142,8 +125,9 @@ public async Task<ActionResult> DeleteCohort(DeleteCommitmentViewModel viewModel
...
+                return Redirect(_providerUrlhelper.ProviderCommitmentsLink($"{viewModel.ProviderId}/unapproved/{viewModel.HashedCommitmentId}/details"));
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs:130]: this controller action should no longer be hit should it?

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs (hunk 2 above)
Comment (hkf-tech) [reply src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs:101]: Updated to log before redirection

**Review by cofaulco — APPROVED** [review]

**Pull request merged**